layout/layout/layout-admin/layout-admin-component.js

```diff
@@ -143,18 +151,14 @@ class LayoutAdmin extends PureComponent {
         <Search />
 
         <Modal
-          open={this.state.modalOpen}
+          open={modalOpen}
           options={modal.options}
           loading={modal.loading}
-          toggleModal={this.props.toggleModal}
-          setModalOptions={this.props.setModalOptions}
+          toggleModal={toggleModal}
+          setModalOptions={setModalOptions}
         />
 
-        <Toastr
-          preventDuplicates
-          transitionIn="fadeIn"
-          transitionOut="fadeOut"
-        />
+        <Toastr preventDuplicates transitionIn="fadeIn" transitionOut="fadeOut" />
```

Comment:
You're being inconsistent listing properties, please follow the rule we talked about.